### ISSUE TITLE
[Feature Request] toggle-keys for select with chip-tags

### ISSUE THREAD

**Issue by vanyakosmos** [vuetifyjs/vuetify#3287]
### New Functionality
Create new tag by pressing `space` or `,` or whatever keyCodes are specified. This behaviour needed for situations when only single-word tags allowed. Currently it's only possible to create/apply tag after pressing `enter` or `tab`.

### Improvements
Had the idea that I can catch new letters on input event and then check latest letter and somehow simulate enter press, but `v-select` emits only `change` event which returns newly added tag...

### Bugs or Edge Cases it Helps Avoid
dunno

**Comment by johnleider** [comment]
@vuetifyjs/dev-team thoughts?

**Comment by chewy94** [comment]
I can see this being useful in many situations such as single word items. For `space` to work, I would think we need to create a prop to signal that the select only accepts single word tags

**Comment by bricewge** [comment]
In #1790 @KaelWD said he tried to add this functionality but choose not to. Maybe he can helps us here on what not do when implementing this.

**Comment by KaelWD** [comment]
That was quite a while ago, the relevant code has gone through several iterations since then. I'll have a look to see if it's easily doable now.

**Comment by torarnv** [comment]
I'm facing a similar issue, where I would like to customize the behavior of the select autocompleter:

1. Choosing an item from the autocomplete menu via enter/tab should close the menu.
   I have a solution for this by capturing keydown events (see codepen), but not sure if this
   is the best approach.
2. Writing 'Cal', which gives only one search result, in which case I want enter/tab to autocomplete
   to 'California', not just 'Cal'
3. Except when the menu is then closed via Escape, in which case I want enter/tab to autocomplete
   the partial lazySearch. Right now this results in the menu opening.

https://codepen.io/torarnv/pen/EEwWOv?editors=1010

**Comment by vitor14** [comment]
Any solution for add a tag pressing 'SPACE' ?

**Comment by johnleider** [comment]
We are revisiting this for v1.1

**Comment by KaelWD** [comment]
```vue
<v-autocomplete tags v-model="selected" :delimiters="[', ', ';', ' ']"/>
```
![2018-05-07_02-42-24](https://user-images.githubusercontent.com/16421948/39675570-4a270256-51a0-11e8-8b26-d62c6291455d.gif)

**Comment by virtualadrian** [comment]
**TL;DR:** Here's the code:
[https://codepen.io/virtualadrian/pen/pOeVPX](https://codepen.io/virtualadrian/pen/pOeVPX)

Fair warning: (1) I am new to Vue. (2) I'm not certain if this is what the OP needed, however I was searching for a tagging component. After poking around github, stackoverflow and the docs. I was able to stitch this together:

![image](https://user-images.githubusercontent.com/10678676/44956351-81bd7180-ae90-11e8-9e82-372787efadf9.png)

```html
          <v-combobox multiple
                    v-model="select" 
                    label="Tags" 
                    append-icon
                    chips
                    deletable-chips
                    class="tag-input"
                    :search-input.sync="search" 
                    @keyup.tab="updateTags"
                    @paste="updateTags">
          </v-combobox>
```

```javascript
new Vue({
  el: "#app",
  data() {
    return {
      select: ['add-tags-with', 'enter', 'tab', 'paste'],
      items: [],
      search: "" //sync search
    };
  },
  methods: {
    updateTags() {
      this.$nextTick(() => {
        this.select.push(...this.search.split(","));
        this.$nextTick(() => {
          this.search = "";
        });
      });
    }
  }
});

```

```css
.tag-input span.chip {
  background-color: #1976d2;
  color: #fff;
  font-size: 1em;
}

.tag-input span.v-chip {
  background-color: #1976d2;
  color: #fff;
  font-size:1em;
  padding-left:7px;
}

.tag-input span.v-chip::before {
    content: "label";
    font-family: 'Material Icons';
    font-weight: normal;
    font-style: normal;
    font-size: 20px;
    line-height: 1;
    letter-spacing: normal;
    text-transform: none;
    display: inline-block;
    white-space: nowrap;
    word-wrap: normal;
    direction: ltr;
    -webkit-font-feature-settings: 'liga';
    -webkit-font-smoothing: antialiased;
}
```

**Comment by vuoriliikaluoma** [comment]
Documentation is missing for the `delimiters` attribute.
Had to look at the source to see if this was still a thing.

**Comment by anthonycheng12** [comment]
So is there a way to do this without writing custom methods?

**Comment by vuoriliikaluoma** [comment]
@anthonycheng12
It only works in VCombobox component.
Like so: `<v-combobox tags :delimiters="[',', ';', ' ']"></v-combobox>`
The delimiters property takes and array of strings.
The example uses comma, semi-colon and space.
These delimiters do not replace the normal Tab and Enter keys.
